packages/exporter-trace-otlp-grpc/src/types.ts

```diff
@@ -45,9 +45,15 @@ export interface OTLPExporterConfigNode
   extends otlpTypes.OTLPExporterConfigBase {
   credentials?: grpc.ChannelCredentials;
   metadata?: grpc.Metadata;
+  compression?: CompressionAlgorithm;
 }
 
 export enum ServiceClientType {
   SPANS,
   METRICS,
 }
+
+export enum CompressionAlgorithm {
+  NONE = 0,
+  GZIP = 2
```

Comment:
Why 2? Is this defined by the grpc client we're using?

Reply:
Yes it's defined by the [grpc client](https://github.com/grpc/grpc-node/blob/master/doc/compression.md#client-side).

Reply:
Can you add a comment to make it clear where those numbers came from so nobody changes them in the future?

Reply:
Yes. This is completed.